handlers/misttriggers/push_end.go

```diff
@@ -130,6 +131,13 @@ func (d *MistCallbackHandlersCollection) SegmentingPushEnd(w http.ResponseWriter
 
 	// TODO: Start Transcoding (stubbed for now with below method)
 	stubTranscodingCallbacksForStudio(callbackUrl)
+	
+	si := cache.DefaultStreamCache.Segmenting.Get(streamName)
+	transcodeRequest := handlers.TranscodeSegmentRequest {
+		SourceFile:    si.SourceFile,
+		CallbackUrl:   si.CallbackUrl,
+	}
+	go handlers.RunTranscodeProcess(d.MistClient, transcodeRequest)
```

Comment:
Cool! I think for the `/vod` API behavior, you might want to wrap this into a local helper which handles the results sending the appropriate callbacks to Studio etc. I guess that's somewhat what you've done on your local change that you mentioned tho?

Btw does this `StreamInfo` have the "probing" information of the file? If so, should be really easy to generate the profiles copying that logic from `task-runner`